bin/nef-playground

```diff
@@ -93,6 +88,47 @@ setDependency() {
   fi
 }
 
+##
+#   setPlaygroundPlatform(String projectFolder, String projectName, String podfilePath)
+#   - Parameter `projectFolder`: path to the project folder.
+#   - Parameter `projectName`: the project name.
+#   - Parameter `podfile`
+##
+setPlaygroundPlatform() {
```

Comment:
now we have 2 templates: one of them for iOS platform, another for macOS platform